packages/connect-react/src/components/ControlSelect.tsx

```
@@ -0,0 +1,93 @@
+  options: {label: string; value: T}[]
+  selectProps?: ReactSelectProps
+}
+
+export function ControlSelect<T>({ isCreatable, options, selectProps }: ControlSelectProps<T>) {
+  const formFieldCtx = useFormFieldContext()
+  const { id, prop, value, onChange } = formFieldCtx
+  const { select, theme } = useCustomize()
+
+  const baseSelectProps: BaseReactSelectProps = {
+    styles: {
+      container: (base: {}): CSSObjectWithLabel => ({
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)